build.gradle.kts

```diff
@@ -27,7 +27,7 @@ apply {
     plugin("org.jlleitschuh.gradle.ktlint")
 }
 
-task<Wrapper>("wrapper") {
+tasks.withType(Wrapper::class.java) {
```

Comment:
@Tapchicoma In the newest version of gradle, the correct way to do this in a way that defers configuration is to do:
```kotlin
tasks.withType(Wrapper::class.java).configureEach { // ...
```